backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -210,36 +232,63 @@ private Bundle convertRowToFhirBundle(TestResultRow row, UUID orgId) {
             row.getOrderingProviderId().value,
             new PersonName(
                 row.getOrderingProviderFirstName().value,
-                row.getOrderingProviderLastName().value,
                 row.getOrderingProviderMiddleName().value,
+                row.getOrderingProviderLastName().value,
                 null),
             row.getOrderingProviderPhoneNumber().value,
             providerAddr,
             DEFAULT_COUNTRY,
             row.getOrderingProviderId().value);
 
-    var additionalDeviceData =
-        deviceTypeRepository.findDeviceTypeByModelIgnoreCase(row.getEquipmentModelName().value);
-
     String equipmentUid = null;
     String testKitNameId = null;
     String manufacturer = null;
+    String diseaseName = null;
     UUID deviceId = UUID.randomUUID();
-    if (additionalDeviceData != null) {
-      var diseaseCode = row.getTestPerformedCode().value;
+    var testPerformedCode = row.getTestPerformedCode().value;
```

Comment:
can you please switch the `.value` to the getter `getValue()` in the whole file, looks like the `value`, `header`, and `errors` needs to be private on the `ValueOrError` object